examples/jokes/app/routes/jokes[.]rss.tsx

```diff
@@ -3,10 +3,22 @@ import type { LoaderFunction } from "@remix-run/node";
 import { db } from "~/utils/db.server";
 
 export const loader: LoaderFunction = async ({ request }) => {
+  const url = new URL(request.url);
+  // in the official deployed version of the app, we don't want to deploy
+  // a site with unmoderated content, so we only show users their own jokes
+  // or jokes from kody, which are safe to show to everyone
+  // by using search params, you can still share and view jokes from other users
+  const username = url.searchParams.get("jokester") || "kody";
+
   const jokes = await db.joke.findMany({
     take: 100,
     orderBy: { createdAt: "desc" },
     include: { jokester: { select: { username: true } } },
```

Comment:
We might as well get rid of this `include` to avoid a table JOIN since we already know the username, and update the RSS to use that variable.